src/potentials.h

```diff
@@ -251,22 +404,22 @@ struct RepulsionR3 : public PairPotentialBase {
  * More info: doi:10.1063/1.3186742
  *
  */
-class Hertz : public PairPotentialBase {
+class Hertz : public MixerPairPotentialBase {
   protected:
-    std::shared_ptr<ParametersTable> m; // table w. diameter_ij and energy-strength_ij
+    TPairMatrixPtr hydrodynamic_diameter; // rhe_i + rhe_j
+    TPairMatrixPtr epsilon_hertz; // geometric mean of epsilon_i and epsilon_j
+    void initPairMatrices() override;
+
   public:
-    Hertz(const std::string &name = "hertz");
+    Hertz(const std::string &name = "hertz")
+        : MixerPairPotentialBase(name) {};
     inline double operator()(const Particle &a, const Particle &b, const Point &r) const override {
         double r2 = r.squaredNorm();
-        if (r2 <= m->hd(a.id, b.id))
-            return m->ehe(a.id, b.id) * pow((1 - (sqrt(r2) / m->hd(a.id, b.id))), 2.5);
+        if (r2 <= (*hydrodynamic_diameter)(a.id, b.id))
```

Comment:
I dislike `hydrodynamic`. What about just using `sigma` as for LJ, WCA, Hardsphere?

Reply:
The resolution of the Hertz potential is already prepared in the next (queued) pull request.